### DISCUSSION TITLE
rss feeds

### DISCUSSION THREAD

**Discussion by caywin25** [searxng/searxng#1760] [Q&A] [answered]
Hi there

I don't know if this question has already been asked. Sorry if that's the case.

Is it possible to generate an rss feed from a page?
For example, I search for a word from a youtube search page:

!yt foo bar

Is it possible to generate an rss feed from the result of the !yt foo bar page?

Thanks

**Comment by allendema** [comment, marked as answer]
The instance must have enabled acces for rss feeds. 
Also from the preferences > privacy > set HTTP Method to "get".

Then to get generate an rss feed append ```&format=rss``` to the url.

https://yourinstance.tld/search?q=!yt%20foo%20bar&categories=none&format=rss


[1] https://github.com/searxng/searxng/blob/master/searx/settings.yml#L48,L51

**Reply by caywin25** [reply]
Great !!! Thank you very much !
